nixos/modules/services/networking/stunnel.nix

```
@@ -16,8 +16,12 @@ let
   serverConfig = {
     options = {
       accept = mkOption {
-        type = types.int;
-        description = "On which port stunnel should listen for incoming TLS connections.";
+        type = types.either types.str types.int;
+        description = ''
+	  On which [host:]port stunnel should listen for incoming TLS connections.
+	  Note that unlike other softwares stunnel ipv6 address need no brackets,
+	  so to listen on all IPv6 addresses on port 1234 one would use ':::1234'.
+	'';
```

Something like this should work (sorry, if this might not apply, GitHub's UI is not very intuitive for this)
```suggestion
          On which [host:]port stunnel should listen for incoming TLS connections.
          Note that unlike other softwares stunnel ipv6 address need no brackets,
          so to listen on all IPv6 addresses on port 1234 one would use ':::1234'.
        '';
```